### PR TITLE
Bug/0.17/hamburger bug

### PR DESCRIPTION
**Pull request by tristan-88**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

- The menu was not showing up when you click the hamburger/3 vertical dots in smallerscreen

## Related Tickets & Documents

_Link user story from projects.digitalaidseattle.org_

- Related Issue # https://projects.digitalaidseattle.org/project-tasks-details?recordId=recjI9rENCrI9e0MB
- Closes # 06/26/2024

### Medium Screen

Screenshot 2024-06-25 at 10 13 08 PM

### Small Screen

Screenshot 2024-06-25 at 10 15 19 PM

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [x] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Color contrast tested?

## Added/updated tests?
_We are aiming for 100% statement coverage._

- [ ] Yes
- [X ] No, and this is why: It did not need a test cover over small minor changes on a specific viewpoint. At least the unit test goes since these buttons were tested for existence on earlier PRs
- [ ] I need help with writing tests

## Are there any post-deployment tasks we need to perform?
Not at the moment.
## Blastoise
alt_text

### CODE REVIEW

**Comment by DillonOLeary** [comment]
Screenshot 2024-06-27 at 1 27 12 PM
I'm running into this error when I launch it. This is unrelate to your change right?

**Comment by DillonOLeary** [comment]
OMG I love the dedication too hahaha

**Comment by tristan-88** [comment]
> Screenshot 2024-06-27 at 1 27 12 PM I'm running into this error when I launch it. This is unrelate to your change right?

Unrelated to my change, I partnered up with Shareiff and he already fixed that not sure which branch it is on the one he started tables in on, or code clean up, there was a change in DB that caused that.

File: src/components/TicketsTable.tsx
Comment (DillonOLeary) [inline src/components/TicketsTable.tsx:1]: This isn't related to the hamburger bug right?

**Review by DillonOLeary — COMMENTED** [review]
It seems like there's more going on here than just the hamburger bug right?

File: src/components/TicketsTable.tsx
Comment (tristan-88) [reply src/components/TicketsTable.tsx:1]: Nope it isnt

**Comment by tristan-88** [comment]
Oh I see, what happened? I accidentally combined the initial work Shareiff was doing on the tables along with the bug fix I did... Well, I'm going to close this pr and delete the branch. I will just include the bug fix with my up PR for the tables.

**Pull request closed without merging**